### PR TITLE
reorganized some tests files from e2e to integration

### LINKED ISSUE #1680: 🚧 Dev ➾ Reorganize the test directories to better distinguish unit from integration from E2E
### 🔗 Epic|Feature Link

_No response_

### 🚥 Definition of Done

- [ ] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation Complete
- [ ] ♾️ PR Merged
- [ ] 🏆 Product Acceptance

### 🧰 Work Description

Several e2e tests should be integration tests and two integration tests should be unit tests.

### ✅ To Do

1. move the two integration tests into the unit test folder
2. ensure the tests work in the new configuration
3. move the e2e tests that should be integration tests into the integration test directory, rewriting, if required, some of the e2e tests and test utils to reflect the new directory structure.
4. ensure the tests work correctly in the new  directory configuration
5. revise the  ~/taqueria/tests/package.json scripts to suit the new directory structure and file arrangement

### ⚖️ Acceptance Criteria

✨ Functional: 
- [ ] working tests in a new directory configuration
- [ ] 

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- ... 
 
 details of specific tests - this is structural refactor

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by michaelkernaghan**
# 🌮 Taqueria PR

## 🪁 Description

Some tests in E2E are long-running and involve features or actions below the level of experience of a user, These tests are not suited to an E2E test suite, which is to be a succinct proof of acceptance and can be managed in the integration test suite, utilized as a test research tool to refine tests and work out bugs. 

A priority for revising the E2E tests suite to use the CLI-test-Library is to make the tests modular and decomposable so that they may be readily replicated and can be moved from test suite to test suite. Some current E2E tests, notably the taquito-flextesa and the jest plugin tests,  are accompanied by complex dependencies around npm modules or invoke sequences of sandbox and docker manipulations.  While these tests continue to pass, they can await revision by residing among the integration tests.

close #1680 

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [ ] ⛱️ I have completed this PR template in full and updated the title appropriately
- [ ] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🐚 Required changes to the VScE have been made
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🛩️ Summary of Changes

I moved some e2e test files to integration and temporally copied the util folder to integration; it will eventually be removed from e2E.

## 🎢 Test Plan
working tests

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [ ] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [x ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [ ] 🚀 Release ➾

### CODE REVIEW

File: tests/integration/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import { TezosToolkit } from '@taquito/taquito';
+import { exec as exec1 } from 'child_process';
+import fsPromises from 'fs/promises';
+import path from 'path';
+
+export const exec = (cmd: string, opts: Record<string, unknown> = {}): Promise<{ stdout: string; stderr: string }> =>
+	new Promise((resolve: Function, reject: Function) => {
+		exec1(cmd, opts, (err, stdout, stderr) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply tests/integration/utils/utils.ts:8]: will try to fix this

File: tests/integration/utils/utils.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	// The core plugin gets installed automatically when a taqueria project is initialized.
+	// However, the version of this plugin is pulled from npmjs.com, which might be using
+	// an older version of the SDK, and that older version of the SDK might be using a version
+	// of @taqueria/protocol package that is incompatible with the development version of the
+	// taq binary, which is the version that was meant to be used by the E2E tests.
+	//
+	// To fix this, we need to do the following:
+	//
+	// 1) Uninstall the core plugin pulled from npmjs.com
+	await exec('npm uninstall @taqueria/plugin-core', { cwd: projectPath });
```
Comment (jchenche) [inline tests/integration/utils/utils.ts:57]: Don't need this as CORE is no longer installed on `taq init` without a `--workflow` option

Comment (michaelkernaghan) [reply tests/integration/utils/utils.ts:57]: The revised tests are not dependant on the utils file, which will be eventually dropped

Comment (michaelkernaghan) [reply tests/integration/utils/utils.ts:57]: But maybe this is an excellent opportunity to clean it up for the other tests not revised yet. I will try it.

**Review by jchenche — APPROVED** [review]

**Pull request merged**